packages/firestore/src/util/bundle_builder_validation_utils.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Validates that 'value' is a string.
+ *
+ * @private
+ * @internal
+ * @param arg The argument name or argument index (for varargs methods).
+ * @param value The input to validate.
+ * @param options Options that specify whether the string can be omitted.
+ */
+export function validateString(arg: string | number, value: unknown): void {
```

Comment:
it's not clear to me if any of the functions in this file are used or will be used?

Reply:
Oh odd, I thought I had removed this source file. I must have messed up the git push.

Reply:
Fixed.